### ISSUE TITLE
Minecraft 1.10.2 error running installer

### ISSUE THREAD

**Issue by xlilcasper** [Shuudoushi/OpenOS_Plus#62]
Using the direwolf20 pack for 1.10.2 and when I try to install using the pastebin or the installer script from github I get an error on line 32. 

Unexpected symbol near 'in'

Fully upgraded t3 case. t3 graphics, two t3 memory, t3 hard drive, internet card and a wireless card.

**Comment by jonasgeiler** [comment]
I'm using FTB Lite 1.10.2 and getting the same error...

**Comment by bonnedav** [comment]
Same here on HermitPack for 1.10.2.

**Comment by CorneliusCornbread** [comment]
Seriously it's been a year an this hasn't been so much as acknowledged by the devs?

**Comment by CaitlynMainer** [comment]
Shuu is/was active duty Army, hasn't been on IRC in over 2 years... honestly SecureOS may be the last thing he is worried about at this point.  I've tried contacting him a few times with no success...  I don't know enough lua to pick up this project even though it's included in one of my upstream projects so trust me I'm feeling this too.

**Comment by SuPeRMiNoR2** [comment]
I should have some time to take a look at this, I will see if I can at least fix it so it is not totally broken.

**Comment by CaitlynMainer** [comment]
Thank you Super.

**Comment by RedstoneParkour** [comment]
Well i looked into it and here it goes wrong:
```lua
if not fs.exists("/tmp/.install") then
  in = io.open("/tmp/.install", "w")
  in:write(os.date())
  in:close()
end
```
This means that @Shuudoushi in commit 2ad61ce30a50e268631fdbb0a86d3f240bd09cf4 used the word "in" (that is used for for-each loops) as a variable name.
So lua searches for "for" in that line. But Lua finds "=", so it throws an error
It got then fixed in commit db4ee55b0a4e93c8af2d29ddbb148a341cfb4068 (Couldnt mention author), but that then got reverted by @SuPeRMiNoR2 in commit 0e8dc059165bac72531ff1cbbbab4ba31034f66d
